haiku-apps/genio/genio-3.1.recipe

```diff
@@ -31,6 +31,7 @@ PROVIDES="
 REQUIRES="
 	haiku${secondaryArchSuffix}
 	gcc${secondaryArchSuffix}_syslibs_devel
+	noto_emoji${secondaryArchSuffix}
```

Comment:
```suggestion
	noto_emoji$secondaryArchSuffix
```

Reply:
Actually, `noto_emoji` is an any-arch package, so it doesn't have $secondaryArchSuffix at all.

Reply:
In that case, should I remove the secondary arch part?

Reply:
yes:

```suggestion
	noto_emoji
```